builds/misc/images-release.yaml

```diff
@@ -299,16 +299,19 @@ jobs:
           imageName: azureiotedge-agent
           name: "Edge Agent"
           project: Microsoft.Azure.Devices.Edge.Agent.Service
+          version: $(version)
```

Comment:
Does this need to be ${{ parameters.version }} ?

Reply:
That depending on where the `$(version)` is being referred.
In this spot, the `${{ parameters.version }}` is actually being set to the value of `$(version)` from the build pipeline's variable. (They are 2 different variables).